### ISSUE TITLE
Installing mlconjug

### ISSUE THREAD

**Issue by vectomon** [SekouD/mlconjug#73]
I'm trying to install mlconjug into the latest offical python docker container, but it fails.
Here is the command that fails:

Command "/usr/local/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-90fg7720/scikit-learn/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-abm6cqy_/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-90fg7720/scikit-learn/

1. docker pull python
2. docker run -it python bash
3. pip install mlconjug

 - OS: Win10
 - Docker 2.0.0.3, linux VM

I also had problems on my host machine installing it, so this is why I switched to the docker container :)
I've also tried to clone the github repository on my host machine (Win10) and install it from there with `python setup.py install`
But I noticed that it tries to fetch scikit-learn 0.19.x instead of 0.20.2.
So I changed this requirement and now it works, but something is not right.
I ran the following command: `mlconjug -l en help` and it gave me this result: 

{'imperative': {'imperative present': OrderedDict([('2s :', ''),
                                                   ('1p :', ''),
                                                   ('2p :', '')])},
 'indicative': {'indicative past tense': OrderedDict([('1s', 'ed'),
                                                      ('2s', 'ed'),
                                                      ('3s', 'ed'),
                                                      ('1p', 'ed'),
                                                      ('2p', 'ed'),
                                                      ('3p', 'ed')]),
                'indicative present': OrderedDict([('1s', ''),
                                                   ('2s', ''),
                                                   ('3s', 's'),
                                                   ('1p', ''),
                                                   ('2p', ''),
                                                   ('3p', '')]),
                'indicative present continuous': OrderedDict([('1s 1s', 'ing'),
                                                              ('2s 2s', 'ing'),
                                                              ('3s 3s', 'ing'),
                                                              ('1p 1p', 'ing'),
                                                              ('2p 2p', 'ing'),
                                                              ('3p 3p',
                                                               'ing')]),
                'indicative present perfect': OrderedDict([('1s', 'ed'),
                                                           ('2s', 'ed'),
                                                           ('3s', 'ed'),
                                                           ('1p', 'ed'),
                                                           ('2p', 'ed'),
                                                           ('3p', 'ed')])},
 'infinitive': {'infinitive present': OrderedDict([('', '')])}}

**Comment by SekouD** [comment]
Hi @vectomon how are you?

Thank you for contributing to mlconjug by submitting this bug report 👍 .

I just published [mlconjug v3.3.1](https://pypi.org/project/mlconjug/3.3.1/) which solves your installation issues and scikit-learn versions discrepancies.

Please try using mlconjug v3.3.1 and let me know if it solves your issues.

Cheers!

SekouD.

**Comment by vectomon** [comment]
Thanks for the quick reply and fix, now it's easy to install :)
One more question:
Am I missing something, or the English regular verbs are not generated properly?

**Comment by SekouD** [comment]
Hi @vectomon. There was indeed a regression bug when I updated the model to use scikit-learn 0.20.2 preventing English regular verbs to be properly populated.
I created a new issue #75 to fix this problem.
I should release a new verion of lyrics master fixing this issue by tomorrow or Friday.

Thanks a lot for your feedback, and keep provindung bug reports, enhancement requests, etc...

Cheers,

SekouD.
